ci/test/02_run_container.sh

```diff
@@ -44,18 +44,21 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
   CI_DEPENDS_MOUNT="type=volume,src=${CONTAINER_NAME}_depends,dst=$DEPENDS_DIR/built"
   CI_DEPENDS_SOURCES_MOUNT="type=volume,src=${CONTAINER_NAME}_depends_sources,dst=$DEPENDS_DIR/sources"
   CI_PREVIOUS_RELEASES_MOUNT="type=volume,src=${CONTAINER_NAME}_previous_releases,dst=$PREVIOUS_RELEASES_DIR"
+  CI_BUILD_MOUNT=""
 
-  if [ "$DANGER_CI_ON_HOST_CACHE_FOLDERS" ]; then
+  if [ "$DANGER_CI_ON_HOST_FOLDERS" ]; then
     # ensure the directories exist
     mkdir -p "${CCACHE_DIR}"
     mkdir -p "${DEPENDS_DIR}/built"
     mkdir -p "${DEPENDS_DIR}/sources"
     mkdir -p "${PREVIOUS_RELEASES_DIR}"
+    mkdir -p "${BASE_BUILD_DIR}"
```

Comment:
Should the definition of `BASE_BUILD_DIR` from https://github.com/bitcoin/bitcoin/blob/faf15ef3a78f6ee5ef5fb9468e9de04b5fca6b21/ci/test/03_test_script.sh#L118-L119

be moved before this first usage?

Reply:
> It requires the caller to define it, which is intentional.

Why? What happens if the default value will be used? (especially, if a CI job doesn't care about it)

> Should I add a comment explaining it?

If it helps :)

Reply:
Right. That's why I suggest to move `BASE_BUILD_DIR=${BASE_BUILD_DIR:-$BASE_SCRATCH_DIR/build-$HOST}` before.

Reply:
Let's consider this resolved.